.circleci/config.yml

```diff
@@ -90,7 +95,7 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: composer install -n --prefer-dist
+      - run: composer update --prefer-lowest
```

Comment:
Something looks off here. This is inside the code style job, not the test job? Look at the composer commands that were actually run at:

* The [php-56-lowest](https://circleci.com/gh/turnerlabs/validating-xml-encoder/110?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) job
* The [code-fixer](https://circleci.com/gh/turnerlabs/validating-xml-encoder/106?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) job

Reply:
Oh yes. I'll check on that.